service-discovery/quickstart.yml

```
@@ -0,0 +1,31 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: service-discovery
```

mark this one draft please, so that it doesn't get picked up accidentally.
```suggestion
  name: service-discovery
  draft: true
```